src/objects.rs

```diff
@@ -1322,47 +1322,42 @@ pub struct TicketPrice {
     pub ticket_validity_end: Date,
 }
 
-#[derive(Serialize, Deserialize, Debug)]
-#[cfg(feature = "fare-v2")]
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
 pub struct TicketUse {
+    #[serde(rename = "ticket_use_id")]
     pub id: String,
     pub ticket_id: String,
     pub max_transfers: Option<u64>,
```

Comment:
can you just change all the u64 below bu u32. I don't think u64 is necessary

Reply:
ok done